app/models/reservation_scopes.rb

```diff
@@ -56,32 +42,29 @@ def self.included(base) # rubocop:disable MethodLength, AbcSize
       scope :ends_on_days, lambda { |start_date, end_date|
         where(due_date: start_date..end_date)
       }
-      scope :reserved_on_date, ->(date) { overlaps_with_date(date).finalized }
+      scope :reserved_on_date, lambda { |date|
+        overlaps_with_date(date).not_available
+      }
       scope :for_eq_model, lambda { |eq_model|
-        where(equipment_model_id: eq_model.id).finalized
+        where(equipment_model_id: eq_model.id)
```

Comment:
Wouldn't this also return unprocessed / denied requests in addition to actual reservations? I guess that makes more sense as a **scope** that's intended to be chained with others, but we need to make sure that if we only expected finalized reservations before that we're adding the relevant scopes to those queries.